jwst/resample/resample.py

```diff
@@ -132,15 +134,22 @@ def resample_many_to_many(self):
         Used for outlier detection
         """
         for exposure in self.input_models.models_grouped:
-            output_model = self.blank_output.copy()
+            output_model = self.blank_output  #.copy()
+            output_root = '_'.join(exposure[0].meta.filename.replace(
+                '.fits', '').split('_')[:-1])
+            output_model.meta.filename = f'{output_root}_outlier_i2d.fits'
```

Comment:
Not sure we should be hardwiring file names to use the ".fits" extension. Theoretically data models can be serialized to different formats (fits, ASDF, etc.).

Reply:
The `.fits` extension was replaced with the extension from the original input model 'meta.filename' instead in commit 88cc36f.